phylogenetic/defaults/auspice_config.json

```diff
@@ -1,56 +1,60 @@
-{
-  "title": "Real-time tracking of Zika virus evolution",
-  "maintainers": [
-    {"name": "the Nextstrain team", "url": "https://nextstrain.org/team"}
-  ],
-  "data_provenance": [
-    {
-      "name": "GenBank",
-      "url": "https://www.ncbi.nlm.nih.gov/genbank/"
-    },
-    {
-      "name": "USVI",
-      "url": "https://github.com/blab/zika-usvi/"
-    }
-  ],
-  "build_url": "https://github.com/nextstrain/zika",
-  "colorings": [
-    {
-      "key": "gt",
-      "title": "genotype",
-      "type": "categorical"
-    },
-    {
-      "key": "num_date",
-      "title": "date",
-      "type": "continuous"
-    },
-    {
-      "key": "author",
-      "title": "author",
-      "type": "categorical"
-    },
-    {
-      "key": "country",
-      "title": "country",
-      "type": "categorical"
-    },
-    {
-      "key": "region",
-      "title": "region",
-      "type": "categorical"
-    }
-  ],
-  "geo_resolutions": [
-    "country",
-    "region"
-  ],
-  "display_defaults": {
-    "map_triplicate": true
-  },
-  "filters": [
-    "country",
-    "region",
-    "author"
-  ]
-}
+{
+  "title": "Real-time tracking of Zika virus evolution",
+  "maintainers": [
+    {"name": "the Nextstrain team", "url": "https://nextstrain.org/team"}
+  ],
+  "data_provenance": [
+    {
+      "name": "GenBank",
+      "url": "https://www.ncbi.nlm.nih.gov/genbank/"
+    },
+    {
+      "name": "USVI",
+      "url": "https://github.com/blab/zika-usvi/"
+    }
+  ],
+  "build_url": "https://github.com/nextstrain/zika",
+  "colorings": [
+    {
+      "key": "gt",
+      "title": "genotype",
+      "type": "categorical"
+    },
+    {
+      "key": "num_date",
+      "title": "date",
+      "type": "continuous"
+    },
+    {
+      "key": "author",
+      "title": "author",
+      "type": "categorical"
+    },
+    {
+      "key": "country",
+      "title": "country",
+      "type": "categorical"
+    },
+    {
+      "key": "region",
+      "title": "region",
+      "type": "categorical"
+    }
+  ],
+  "geo_resolutions": [
+    "country",
+    "region"
+  ],
+  "display_defaults": {
```

Comment:
This commit changed this file from CRLF to LF, so GitHub shows every line as changed. The salient changes are:

```diff
   "display_defaults": {
-    "map_triplicate": true
+    "map_triplicate": true,
+    "tip_label": "strain"
   },
+  "metadata_columns": [
+    "strain"
+  ],
   "filters": [
```

Reply:
Yup, also viewable in the [specific commit changes](https://github.com/nextstrain/zika/pull/72/commits/8fb92b53fbd6382b5b4dadaec72d861f9698947a#diff-eec2a07074cc3f264719563e7deca28eea78cc40a224230b135f075f23a687d0)

Reply:
Yeah - totally missed that & was editing my comment when you replied!